src/idl_parser.cpp

```diff
@@ -1794,6 +1812,15 @@ void Parser::MarkGenerated() {
 
 CheckedError Parser::ParseNamespace() {
   NEXT();
+
+  if ((opts.lang_to_generate & IDLOptions::kPython) &&
```

Comment:
Does it make sense to switch to a single file for Python?

Reply:
I would like this feature, but it makes it harder to generate code for testing and isn't in line with the behavior of generators of other languages which base on directory structure, I'll add a flag to turn this feature on.